.changesets/avoid-creating-oban-insert-job-root-span.md

```diff
@@ -0,0 +1,6 @@
+---
+bump: "patch"
+type: "fix"
+---
+
+Avoid creating an Oban insert job as a root span. This should fix an issue where "insert job" actions with little information show up in AppSignal when Oban jobs are inserted from an uninstrumented context.
```

Comment:
```suggestion
Avoid creating an Oban insert job event as a root span. This should fix an issue where "insert job" events with little information show up in AppSignal when Oban jobs are inserted from an uninstrumented context.
```

Reply:
I do mean to say "actions", I think? Unless I am mixing up my AppSignal terminology?

The problem is that they end up being root spans and show up as an independent "insert job" action/sample in the performance panel with no further context, and this means that they count towards the "requests" cap.

Reply:
Thank you @tombruijn!